sdk/core/azure/setup.py

```diff
@@ -9,9 +9,25 @@
 from setuptools import setup
 
 
+message = """
+
+Starting v5.0.0, the 'azure' meta-package is deprecated and cannot be installed anymore.
+Please install the service specific packages prefixed by `azure` needed for your application.
+
+Here's a non-exhaustive list of common packages:
```

Comment:
Please add the other storage packages.

Reply:
I didn't wanted this list to be long on purpose, and also I don't want to mention package file-share which has less usage overall. Thoughts?

Reply:
I would generally okay with only blobs as I believe this makes up ~90% of storage usage.
However it would maybe be good to have something like "for an exhaustive list of available Azure services and SDKs, please see magic-url-to-all-the-sdks.com"

Reply:
Added link to https://aka.ms/azsdk/python/all